Meta/CMake/lagom_compile_options.cmake

```diff
@@ -6,6 +6,7 @@ add_compile_options(-fsigned-char)
 add_compile_options(-ggnu-pubnames)
 if (NOT WIN32)
     add_compile_options(-fPIC)
+    add_compile_options(-D_FILE_OFFSET_BITS=64)
```

Comment:
It seems like the wasm build failed for this PR. I can't check the logs, but I'm assuming it's related to this change.

We build libjs into a wasm module for https://serenityos.github.io/libjs-website/repl/

Is this define guaranteed to do anything on... Linux, macOS, Linux with musl, FreeBSD, OpenBSD, Haiku, ... etc? Or should it be more targeted?